src/actions/write/finalizeOperatorStakeDecrease.ts

```diff
@@ -11,7 +11,7 @@ export const finalizeOperatorStakeDecrease = async (
   const { gateways = {}, balances } = state;
 
   if (!(target in gateways)) {
-    throw new ContractError("This Gateway's wallet is not registered");
+    throw new ContractError('This target is not a registered gateway');
```

Comment:
nit - use a constant and check for the constant in a test

Reply:
This interaction is going away in the future with stateTick

We can update all error messages in the future and align for consistency.